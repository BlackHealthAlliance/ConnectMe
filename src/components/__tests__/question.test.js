import React from "react";
import renderer from "react-test-renderer";
import Question from '../question';
describe('Question', () => {
  
  test('It should render a question', () => {

    const mockOnClick = jest.fn();
    const tree = renderer
      .create(<Question
        id="test"
        headerText="Test question?"
        optionsArray={["option1", "option2"]}
        next=""
        onClickHandler={mockOnClick}/>)
      .toJSON()
    expect(tree).toMatchSnapshot();
  });

});