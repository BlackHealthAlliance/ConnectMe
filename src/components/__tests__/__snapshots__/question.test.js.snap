// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question It should render a question 1`] = `
<div
  className="question"
>
  <h1>
    Test question?
  </h1>
  <button
    className="choiceButton"
    onClick={[Function]}
  >
    option1
  </button>
  <button
    className="choiceButton"
    onClick={[Function]}
  >
    option2
  </button>
  <br />
  <button
    className="skipButton"
    onClick={[Function]}
  >
    Skip
  </button>
</div>
`;
